### DISCUSSION TITLE
display image in clolumn

### DISCUSSION THREAD

**Discussion by ashek1412** [Power-Components/livewire-powergrid#732] [General]
Hi,
Is it possible to display an  image in the column like below
![image](https://user-images.githubusercontent.com/18645594/202768880-22f49cda-8780-4d1b-b963-5c0ac248f302.png)

**Comment by BenOussama180** [comment]
yes just put : 
`Blade::render('<img src="xxxx"   class="custom_class" />)`
in the return callback for the column
